semgrep-core/src/parsing/Parse_pattern.ml

```diff
@@ -130,7 +130,7 @@ let parse_pattern lang ?(print_errors = false) str =
         let res = Parse_hack_tree_sitter.parse_pattern str in
         extract_pattern_from_tree_sitter_result res print_errors
     (* use adhoc parser (neither pfff nor tree-sitter) *)
-    | Lang.Yaml -> Yaml_to_generic.any str
+    | Lang.Yaml -> Yaml_to_generic.yaml_any str
```

Comment:
this is not consistent with what we do for the other languages. I'd rather keep the any() and program() entry points.

Reply:
oh ok. Not sure I understood but yes adding a comment would help!